packages/babel-traverse/src/path/conversion.js

```diff
@@ -461,8 +461,12 @@ function getThisBinding(thisEnvFn, inConstructor) {
         if (supers.has(child.node)) return;
         supers.add(child.node);
 
-        child.replaceWith(
-          t.assignmentExpression("=", t.identifier(thisBinding), child.node),
+        child.insertAfter(
```

Comment:
See [this comment](https://github.com/babel/babel/issues/9020#issuecomment-440185493) for why I made this change